andela/zinnia-ah-backend
# #164955297 setup migrations for project

#### What does this PR do?
create the migrations for this project

#### Description of Task to be completed?
- [x] define user model and migration
- [x] define article model and migration
- [x] define comment model and migration
- [x] define report model and migration
- [x] define article model and migration
- [x] define article-rating model and migration
- [x] define user-bookmarks model and migration
- [x] define user-follow migration
- [x] define comment-like model and migration 
- [x] define article likes and dislike model and migration 

#### How should this be manually tested?
* clone `https://github.com/andela/zinnia-ah-backend.git`
* checkout to  the branch = `ch-setup-migrations-164955297`
* run `npm install && install -g pg`
* run `cp .env.example .env`
* run `sequelize db:migrate`
* Check your postgres database for all the tables

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?(if applicable)
#164955297

#### Screenshots
None
